docs/deprecations.rst

```diff
@@ -80,6 +80,16 @@ A number of constants have been deprecated and will be removed in Pillow 10.0.0
     was reversed in Pillow 9.4.0 and those constants will now remain available.
     See :ref:`restored-image-constants`
 
+PSFile
+~~~~~~
+
+.. deprecated:: 9.4.0
```

Comment:
The next release is 9.5.0:

```suggestion
.. deprecated:: 9.5.0
```

Also the deprecations are listed in the order of the deprecation version, so let's move this bit down to the end. 

Similarly, move the info from `docs/releasenotes/9.4.0.rst` to `docs/releasenotes/9.5.0.rst`